oh-my-bash.sh

```diff
@@ -144,6 +144,14 @@ unset -v _omb_init_files _omb_init_file
 # Load the theme
 if [[ $OSH_THEME == random ]]; then
   _omb_util_glob_expand _omb_init_files '"$OSH"/themes/*/*.theme.sh'
+
+  # Remove ignored themes from the list
+  for _omb_ignore_theme in random "${OMB_THEME_RANDOM_IGNORED[@]}"; do
```

Comment:
Could you rename those variables to have the prefix `_omb_init_`? Such as `_omb_init_theme` or `_omb_init_ignore_theme`.